### DISCUSSION TITLE
lightspark master latst git hung computer entirely with nouveau drivers

### DISCUSSION THREAD

**Discussion by estatistics** [lightspark/lightspark#903] [General]
lightspark master latst git hung computer entirely with nouveau driver

I run the smallest game swf that i had. Game run very smoothly, until cright click for menu. computer hung with nouveau drivers. 

`kernel rejected pushbuf: there is no such file or directory. `

if you like to do specific test tell me!

![IMG_20220216_230242](https://user-images.githubusercontent.com/41580704/154358806-29d74e02-dfa4-40de-b3bc-9f9ad8a0df5e.jpg)

![IMG_20220216_230223](https://user-images.githubusercontent.com/41580704/154358786-303ab782-4c2f-497b-8847-a32049b121f9.jpg)

**Comment by dbluelle** [comment]
I don't have any nVidia card, so I can't reproduce that.
Does this happen on every swf file you tested?
Maybe you can test one of the swf files that is attached to one of the open issues (see https://github.com/lightspark/lightspark/issues)
